docs/security.rst

```diff
@@ -13,7 +13,7 @@
 
 For more details, check the :ref:`configuration options<schemaregistry_config>`.
 
-If you want to see full configurations for a secured |sr|, please refer to the :ref:`Confluent Platform demo<tutorials>`
+If you want to see full configurations for a secured |sr|, please refer to the :ref:`Confluent Platform demo<cp-demo>`
```

Comment:
add a space between the title and anchor reference (see http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role). i think it builds okay externally, but fails on the local build - and the best practice is to include the space.